packages/elements-dev-portal/src/components/NodeContent/index.tsx

```diff
@@ -39,6 +39,7 @@ export const NodeContent = ({ node, Link, hideTryIt, hideMocking }: NodeContentP
                 nodeData={node.data}
                 nodeTitle={node.title}
                 hideTryIt={hideTryIt}
+                useNodeForRefResolving
```

Comment:
Shouldn't this be true by default? 🤔

Reply:
I am ok either way. 🤷

Reply:
I'm for making it true by default. Is there a use case for not resolving refs?

Reply:
There must be as @mpodlasin  added it as a prop and not by default. @mpodlasin do you know the reason maybe?